### ISSUE TITLE
occupations plot edits

### ISSUE THREAD

**Issue by lizgzil** [nestauk/ojd_daps_skills#168]
- [x] get rid of "Other"
- [ ] group decision on how to group the other occs

**Comment by lizgzil** [comment]
https://docs.google.com/spreadsheets/d/1R1bkAZRz1gOx65D0qxf2moRNWVN5liBRfI4JXv-H5P0/edit#gid=1261644232

**Comment by india-kerle** [comment]
Pulling this out also from cath: 

<img width="442" alt="Screenshot 2023-01-26 at 16 21 38" src="https://user-images.githubusercontent.com/46863334/214890420-fcc68063-7eaa-4354-b1cd-4ff0e66e4aa5.png">

- Do you think we should change 'number of job adverts' to 'number of job adverts for this occupation' just to make it super clear. 
- 'mid similarity'  doesn't quite sound right. Perhaps 'somewhat similar'
- 'Select an occupation'
- How can we make the 'Any' option clearer? 
- 'Percentage of job adverts that mention this skill at least once' (if that's right!)
- Could we make the blue bar area smaller so there's more space for the skill names?

**Comment by lizgzil** [comment]
I've addressed most of these in https://github.com/nestauk/ojd_daps_skills/pull/171/commits/ac75e7c3aa30926b1fa9bf9ce05d123d5564abb2 apart from :
- How can we make the 'Any' option clearer?

On "Could we make the blue bar area smaller so there's more space for the skill names?" <- I've kind of done this, but found it a bit tricky . where the y line sits is dynamic to how much space the y labels take up (e.g. if they are all short skills then the y axis will be over to the left), but I have increased its maximum spacing, so it does seem to fit in the full skill label now.
![Screenshot 2023-01-26 at 18 09 15](https://user-images.githubusercontent.com/15956065/214916633-0703a602-9433-4c27-91cd-31cf51f61f25.png)
